### DISCUSSION TITLE
Uploading models to be approved via PR and metrics displayed in plot

### DISCUSSION THREAD

**Discussion by ruaridhg** [SciKit-Surgery/cmicHACKS2#30] [Ideas]
People can add their model to the repo via pull request that will be approved if it meet the requirements:

- Augmentations described with python package used
- Dataset described with link?
- Hyperparameters.json file
- checkpoint.pth file
- epochs, training and test loss file
- Evaluation_metrics.json file

With all this information, a script or notebook can then read in the evaluation metrics and plot each evaluation metric vs epochs in suplots e.g. 6 metrics can be displayed in 6 subplots in a 2x3 grid

Examples to be plotted:
"Jaccard": jaccard,
"F1": f1,
"Recall": recall,
"Precision": precision,
"Acc": acc,
"F2": f2,
"Mean FPS": mean_fps,

Loss vs epochs
